ci/windows/build_cub.ps1

```diff
@@ -1,4 +1,4 @@
-Param(
```

Comment:
These seemingly empty changes fix the BOM to UTF-8.

Reply:
Nice, I wondered already yesterday whether it was an encoding thing!